spec/models/comment_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  it 'correctly created' do
+    comment = Comment.new(user: User.new, post: Post.new, content: 'Test')
+    expect(comment).to be_valid
+  end
+
+  it 'when no content presence' do
```

Comment:
You can think the string passed to `it` as an explanation on what's expected, e.g. in this case `returns false`... Note that:
* If you do this change, you won't know where the `false` is returned, for that you could use extra `describe` to tell the developer where it's being returned as mentioned in https://www.betterspecs.org/#describe. 
* If you do this change and also add the new `describe` as mentioned before, you still won't know when the `false` is returned, so you can add a `context` https://www.betterspecs.org/#contexts to tell de developer the scenario the test is in (in this case the string you are using seems like a good example for `context`)

Taking these 2 points into consideration, would you mind updating the rest of specs?